apps/fluent-tester/src/TestComponents/Text/TextTest.tsx

```diff
@@ -30,7 +30,7 @@ export const TextTest: React.FunctionComponent = () => {
     uwpStatus: 'Experimental',
     iosStatus: 'Experimental',
     macosStatus: 'Experimental',
-    androidStatus: 'Beta',
+    androidStatus: 'Experimental',
```

Comment:
This is the wrong test folder for Text V1. Please look at TextExperimental instead.